packages/react-components/react-tree/src/contexts/treeContext.ts

```diff
@@ -25,12 +26,19 @@ export type TreeItemRequest = { itemType: TreeItemType } & (
   | OmitWithoutExpanding<TreeCheckedChangeData, 'selectionMode' | 'checkedItems'>
 );
 
-// helper type that avoids the expansion of unions while inferring it, should work exactly the same as Omit
+/**
+ * helper type that avoids the expansion of unions while inferring it,
+ * should work exactly the same as Omit
+ */
 type OmitWithoutExpanding<P, K extends string | number | symbol> = P extends unknown ? Omit<P, K> : P;
 
-const defaultContextValue: TreeContextValue = {
-  treeType: 'nested',
+/**
+ * @internal
+ */
+export const defaultTreeContextValue: TreeContextValue = {
```

Comment:
Is export is needed there?

Reply:
This is being spread in the `useSubFlatTree` state return, so yes, this has to be exported.